packages/react-headless/src/types/INotificationsStoresCollection.ts

```diff
@@ -1,6 +1,6 @@
-import { IRemoteNotification } from '../types';
-import INotificationRepository from './INotificationRepository';
-import INotificationStore from './INotificationStore';
+import { IRemoteNotification } from '../types/index.js';
```

Comment:
💅 I think it's better to directly import here, instead of going through `index.ts`, as we're in the same package.

```suggestion
import IRemoteNotification from './IRemoteNotification.js';
```

Reply:
Agree, we should drop all barrel files. They're bad for bundlers / tree shaking. I didn't do it here, as I simply added the extensions using https://www.npmjs.com/package/fix-esm-import-path.